examples/movie_view_ratings.py

```diff
@@ -52,24 +52,29 @@ def calc_dp_rating_metrics(movie_views, ops, public_partitions):
     dp_engine = pipeline_dp.DPEngine(budget_accountant, ops)
 
     # Specify which DP aggregated metrics to compute.
-    params = pipeline_dp.AggregateParams(metrics=[pipeline_dp.Metrics.COUNT],
-                                         max_partitions_contributed=2,
-                                         max_contributions_per_partition=1,
-                                         low=1,
-                                         high=5,
-                                         public_partitions=public_partitions)
-
-    # Specify how to extract is privacy_id, partition_key and value from an
+    params = pipeline_dp.AggregateParams(
+        noise_kind=pipeline_dp.aggregate_params.NoiseKind.LAPLACE,
```

Comment:
Nit: pipeline_dp.NoiseKind.LAPLACE  (it's already in __init__.py)

Reply:
👍 Thank you!